ice/env.zep

```diff
@@ -0,0 +1,80 @@
+
+namespace Ice;
+
+use Ice\Config\Ini;
+use Ice\Exception;
+
+/**
+ * Environment helper.
+ *
+ * @package     Ice/Config
+ * @category    Configuration
+ * @author      Ice Team
+ * @copyright   (c) 2014-2018 Ice Team
+ * @license     http://iceframework.org/license
+ */
+class Env extends Ini
+{
+
+    /**
+     * Env constructor.
+     *
+     * @param string data Path to the env file
+     */
+    public function __construct(var data = null)
+    {
+        // No support for passing variables by reference yet zephir/issues/203
+        //let this->data = &_ENV;
+
+        parent::__construct(data);
```

Comment:
`php` config in favor of `ini`, which has to load config from an ini file.
```php
class Env extends Config
{
    public function __construct(var data = null)
    {
        parent::__construct(data);

        let _ENV = array_merge(_ENV, this->data);
    }
}

Reply:
But on `.env` file you can put your environment variables as key-value pairs
```ini
DB_NAME=database name
DB_PASSWORD=@$fq42$d2r2
DB_HOST="localhost"
```
syntax is more similar to ini, so I used that adapter.

Reply:
for this case `$config = new Env((new Ini('env.ini'))->getData())`

Reply:
Right, but not env.ini, but .env, like https://github.com/vlucas/phpdotenv